components/gridlayout/overview.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Overview
+page_title: GridLayout Overview
+description: Overview of the GridLayout for Blazor.
+slug: gridlayout-overview
+tags: telerik,blazor,gridlayout,overview
+published: True
+position: 0
+---
+
+# GridLayout Overview
+
+The <a href="https://www.telerik.com/blazor-ui/gridlayout" target="_blank">Blazor GridLayout component</a> allows you to arrange the contents of the component in rows and columns in a grid structure. 
+
+#### In This Article:
+
+* [Basics](#basics)
+
+* [Features](#features)
+
+## Basics
+
+#### To use a Telerik GridLayout for Blazor
+
+1. Declare the `<TelerikGridLayout>` tag 
+
+1. To define columns in the GridLayout add one or more instances of the `<GridLayoutColumn>` to the `<GridLayoutColumns>` collection.
+
+    * The number of `<GridLayoutColumn>` should be the same as the number of the `<GridLayoutItem>`.
+    
+1. To define rows in the GridLayout add one or more instances of the `<GridLayoutRow>` to the `<GridLayoutRows>` collection.
+
+1. Provide one or more instances of the `<GridLayoutItem>` to the `<GridLayoutItems>` collection.
+
+>caption A basic configuration of the Telerik GridLayout
+
+````CSHTML
+@* Split the items in the GridLayout in three columns and two rows. *@
+
+<TelerikGridLayout>
+    <GridLayoutColumns>
+        <GridLayoutColumn Width="200px"></GridLayoutColumn>
+        <GridLayoutColumn Width="200px"></GridLayoutColumn>
+        <GridLayoutColumn Width="200px"></GridLayoutColumn>
+    </GridLayoutColumns>
+    <GridLayoutRows>
+        <GridLayoutRow Height="50px"></GridLayoutRow>
+    </GridLayoutRows>
+    <GridLayoutItems>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 1
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 2
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 3
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 4
+            </div>
+        </GridLayoutItem>
+        <GridLayoutItem>
+            <div style="border: 1px gray solid;">
+                item 5
+            </div>
+        </GridLayoutItem>
+    </GridLayoutItems>
+</TelerikGridLayout>
+````
+
+>caption The result from the code snippet above
+
+![overview of the GridLayout functionality](images/gridlayout-overview-basic-example.png121)
+
+
+>caption Component namespace and reference
```

Comment:
This second snippet brings little value, but expands the article length. It can be merged with the first one.

Reply:
This is a common practice across our documentation, to add a snippet that showcases the namespace and reference. I would rather leave it there.